cairo.html.markdown

```diff
@@ -0,0 +1,714 @@
+---
+language: Cairo
+filename: learnCairo.sol
+contributors:
+  - ["Darlington Nnam", "https://github.com/Darlington02"]
+---
+
+# Cairo
+Cairo is StarkNet's native language and the first Turing-complete language for scripting provable programs (where one party can prove to another that a certain computation was executed correctly) for general computations.
+# StarkNet
+StarkNet is a decentralized ZK-rollup that operates as an Ethereum layer 2 chain. StarkNet enables Decentralized applications to achieve unlimited scale for their computation - without compromising Ethereum's decentralization and security, thereby solving the Scalability Trilemma.
```

Comment:
This just sounds like marketing

Reply:
Wasn't really the aim, but I'd update it right now, thanks :)